tools/vagrant/vagrant-nfs.md

```diff
@@ -39,6 +39,20 @@ Vagrant.configure("2") do |config|
 end
 ```
 
+One of the common errors when using NFS this way might be that your NFS version does not support UDP protocol which gets added by Vagrant as a default. 
+
+The error says `mount.nfs: requested NFS version or transport protocol is not supported`.
+
+In that case it's enough to disable UDP with `nfs_udp` option:
+
+```
+Vagrant.configure("2") do |config|
+  # ...
+
+  config.vm.synced_folder ".", "/vagrant", type: "nfs", nfs_udp: false
```

Comment:
> config.vm.synced_folder ".", "/vagrant", type: "nfs", nfs_udp: false

Would not this line simply suffice? Is it worth repeating `Vagrant.configure("2") do |config|`?
WDYT?
￼